test/Profiler/coverage_relative_path.swift

```diff
@@ -0,0 +1,18 @@
+// %s expands to an absolute path, so to test relative paths we need to create a
+// clean directory, put the source there, and cd into it.
+// RUN: rm -rf %t
+// RUN: mkdir -p %t/foo/bar/baz
+// RUN: cp %s %t/foo/bar/baz/coverage_relative_path.swift
+// RUN: cd %t/foo/bar
+
+// RUN: %target-swift-frontend -profile-generate -profile-coverage-mapping -emit-ir baz/coverage_relative_path.swift | %FileCheck -check-prefix=ABSOLUTE %s
+//
+// ABSOLUTE: @__llvm_coverage_mapping = {{.*"\\01.*foo.*bar.*baz.*coverage_relative_path\.swift}}
+
+// %clang_cc1 -fprofile-instrument=clang -fcoverage-mapping -emit-llvm -mllvm -enable-name-compression=false -main-file-name debug-dir.cpp baz/debug-dir.cpp -fdebug-compilation-dir . -o - | FileCheck -check-prefix=RELATIVE %s
```

Comment:
Any reason to keep the clang_cc1 invocation around?

Reply:
Sorry I was using this as reference and didn't mean to add it here, removed